settings.gradle

```diff
@@ -52,6 +52,7 @@ include "jackson-databind"
 include "json-core"
 include "management"
 include "messaging"
+include 'netty-graal'
```

Comment:
Sorry to be picky here but we are using double-quotes `"` instead of single-quote `'` :see_no_evil: